pyinstaller/specterd.spec

```diff
@@ -33,16 +33,16 @@ elif platform.system() == 'Linux':
                 break
     packaged_software_datas = [
         ('./torbrowser/tor-browser-linux64-10.0.15_en-US.tar.xz', 'torbrowser'),
-        ('./bitcoind/bitcoin-0.21.0-x86_64-linux-gnu.tar.gz', 'bitcoind'),
-        ('./bitcoind/bitcoin-0.21.0-arm-linux-gnueabihf.tar.gz', 'bitcoind'),
+        ('./bitcoind/bitcoin-0.21.1-x86_64-linux-gnu.tar.gz', 'bitcoind'),
+        ('./bitcoind/bitcoin-0.21.1-arm-linux-gnueabihf.tar.gz', 'bitcoind'),
```

Comment:
I think we could package only one version of bitcoin core here depending on `platform.processor()`

Reply:
But then we'll have to make 2 different Linux builds on 2 different Linux machines.

Reply:
But we need to do it anyway. Currently we release only for x86_64, for arm we need another build machine

Reply:
Right, my bad. I'll update it

Reply:
Updated